{
  "Header": {
    "Original": "CVE-2017-18359 (PostGIS 2.x before 2.3.3, as used with PostgreSQL, allows remote attac ...)",
    "Line": 303009,
    "ID": "CVE-2017-18359",
    "Description": "(PostGIS 2.x before 2.3.3, as used with PostgreSQL, allows remote attac ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-2857-1 DLA-1653-1}",
      "Line": 303010,
      "Type": "xref",
      "Bugs": [
        "DLA-2857-1",
        "DLA-1653-1"
      ]
    },
    {
      "Original": "- postgis 2.3.3+dfsg-1 (low)",
      "Line": 303011,
      "Type": "package",
      "Package": "postgis",
      "Kind": "fixed",
      "Version": "2.3.3+dfsg-1",
      "Severity": "low"
    },
    {
      "Original": "NOTE: https://trac.osgeo.org/postgis/ticket/3704",
      "Line": 303012,
      "Type": "NOTE",
      "Description": "https://trac.osgeo.org/postgis/ticket/3704"
    },
    {
      "Original": "NOTE: https://trac.osgeo.org/postgis/changeset/15444",
      "Line": 303013,
      "Type": "NOTE",
      "Description": "https://trac.osgeo.org/postgis/changeset/15444"
    },
    {
      "Original": "NOTE: https://trac.osgeo.org/postgis/changeset/15445",
      "Line": 303014,
      "Type": "NOTE",
      "Description": "https://trac.osgeo.org/postgis/changeset/15445"
    }
  ]
}